src/components/definition-list/definition-list.stories.mdx

```diff
@@ -42,31 +48,75 @@ Definition List most of the time is in use with other components. Look at the ex
   </Story>
 </Preview>
 
+### Action popover and icon support
+
+<Preview>
+  <Story name="action popover and icon support">
+    <Dl>
+      <Dt>
+        <Box paddingTop="4px" display="inline-flex" alignItems="center">
```

Comment:
We don't really need the `display` and `alignItems` props on this when it just has a string as a child